--- camorama-0.20.7/camorama.desktop.in.orig	2019-04-23 20:57:55.897607664 +0200
+++ camorama-0.20.7/camorama.desktop.in	2019-04-23 21:00:56.689961562 +0200
@@ -4,7 +4,7 @@
 Comment=View, alter and save images from a webcam
 Icon=camorama
 Exec=camorama
-Terminal=0
-Categories=GNOME;AudioVideo;Video;
+Terminal=false
+Categories=GTK;GNOME;AudioVideo;Video;
 Keywords=camorama;webcam;viewer;video-effects;
 StartupNotify=true
