--- camorama-0.19/src/camorama-stock-items.h	2007-06-14 16:05:33.000000000 +0200
+++ camorama-0.19.new/src/camorama-stock-items.h	2011-12-10 08:32:09.856317100 +0100
@@ -1,7 +1,7 @@
 #ifndef CAMORAMA_STOCK_ITEMS_H
 #define CAMORAMA_STOCK_ITEMS_H
 
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
--- camorama-0.19/src/camorama-window.c	2011-12-10 08:51:10.552056445 +0100
+++ camorama-0.19.new/src/camorama-window.c	2011-12-10 08:50:52.068287522 +0100
@@ -28,6 +28,7 @@
 # include <config.h>
 #endif
 
+#include <glib.h>
 #include <glib/gi18n.h>
 #include <libgnomeui/gnome-file-entry.h>
 #include "callbacks.h"
--- camorama-0.19/src/glib-helpers.h	2007-06-14 16:05:33.000000000 +0200
+++ camorama-0.19.new/src/glib-helpers.h	2011-12-10 08:33:22.534408498 +0100
@@ -24,7 +24,7 @@
 #ifndef GLIB_HELPERS_H
 #define GLIB_HELPERS_H
 
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
