diff -up camorama-0.19/src/v4l.h~ camorama-0.19/src/v4l.h
--- camorama-0.19/src/v4l.h~	2007-09-16 15:36:55.000000000 +0200
+++ camorama-0.19/src/v4l.h	2011-02-09 22:48:17.317513026 +0100
@@ -12,8 +12,7 @@
 #include <sys/mman.h>
 #include <fcntl.h>
 #include <gtk/gtk.h>
-#include <linux/types.h>
-#include <linux/videodev.h>
+#include <libv4l1.h>
 #include <signal.h>
 #include <png.h>
 #include <glade/glade.h>
